src/libyang2/Makefile

```diff
@@ -2,10 +2,10 @@
 SHELL = /bin/bash
 .SHELLFLAGS += -e
 
-LIBYANG_URL = https://sonicstorage.blob.core.windows.net/debian/pool/main/liby/libyang
+LIBYANG_URL = https://deb.debian.org/debian/pool/main/liby/libyang2
 
 DSC_FILE = libyang2_$(LIBYANG2_FULLVERSION).dsc
-ORIG_FILE = libyang2_$(LIBYANG2_VERSION).orig.tar.gz
+ORIG_FILE = libyang2_$(LIBYANG2_VERSION).orig.tar.xz
```

Comment:
Is the file format change expected?

Reply:
Yes, the file present in repository is [libyang2_2.1.148.orig.tar.xz](https://ftp.debian.org/debian/pool/main/liby/libyang2/libyang2_2.1.148.orig.tar.xz)